vim/whylog/plugin/whylog.vim

```diff
@@ -0,0 +1,19 @@
+function! whylog#Whylog_Action()
+    if !has('python') | !has('byte_offset')
+        finish
+    endif
+python << EOF
+import whylog_vim
+whylog_vim.whylog_action()
+EOF
+endfunction
+
+function! whylog#Whylog_Teach()
+    if !has('python') | !has('byte_offset')
+        finish
+    endif
+python << EOF
+import whylog_vim
```

Comment:
http://stackoverflow.com/questions/5027352/how-to-test-if-one-python-module-has-been-imported